src/dune/node_state.py

```diff
@@ -42,4 +42,8 @@ def string(self, file=sys.stdout, sep=',', simple=True):
             running_str='Y'
         if simple:
             return f"{self.name}{sep}{active_str}{sep}{running_str}{sep}{self.http}{sep}{self.p2p}{sep}{self.ship}"
-        return f"{self.name}\t\t {sep}    {active_str}\t   {sep}    {running_str}     {sep} {self.http} {sep} {self.p2p} {sep} {self.ship}"
+        buffer = ""
+        while (len(self.name) + len(buffer)) < name_width:
+            buffer += " "
+        return \
+            f"{self.name}{buffer}{sep}    {active_str}    {sep}    {running_str}     {sep} {self.http} {sep} {self.p2p} {sep} {self.ship}"
```

Comment:
I missed another `while` improvement in the first review:
```suggestion
        return f"{self.name}{' ' * (name_width - len(self.name))}{sep}    {active_str}    {sep}    {running_str}     " +\
            f"{sep} {self.http} {sep} {self.p2p} {sep} {self.ship}"
```

Reply:
Thanks. Python isn't my first language. I've updated and tested this file as well.